kuma/spam/models.py

```diff
@@ -0,0 +1,18 @@
+from django.conf import settings
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
```

Comment:
I like that too. As long as it is standard across the project so we don't have to double check the alias everywhere.